test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/bad.py

```diff
@@ -0,0 +1,31 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+name: bad
```

Comment:
I love this! Bad lookup! Bad!

Reply:
I copied that name from an existing module in that test suite: https://github.com/ansible/ansible/blob/916d12eb13fba6b2ea1870d133599cfdc1b1cb98/test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/modules/bad.py